goagen/codegen/validation.go

```diff
@@ -260,6 +260,7 @@ func ValidationChecker(att *design.AttributeDefinition, nonzero, required, hasDe
 		"context":   context,
 		"target":    target,
 		"targetVal": t,
+		"string":    att.Type.Name() == "string",
```

Comment:
Small thing: do you think you could use `att.Type.Kind() == design.StringKind` instead ? Thank you!